server/views/temporary-accommodation/booking-search/results.njk

```diff
@@ -9,6 +9,19 @@
 {% set pageTitle = "View bookings - " + applicationName %}
 {% set mainClasses = "app-container govuk-body" %}
 
+{# TODO Move this HTML to the searchByCrnOrNameResult once HTML can be standardised between referrals and bookings #}
+{% set resultsHtml %}
+    {% if (not context.errors.prn) and ((response.data|length > 0) or (not crn)) %}
```

Comment:
🤮  but https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/1038 simplifies the condition to only display results if there are results